audit-connector/src/test/java/com/sequenceiq/cloudbreak/audit/converter/AttemptAuditEventResultToGrpcAttemptAuditEventResultConverterTest.java

```diff
@@ -68,22 +93,22 @@ void convertWithResultServiceEventData() {
                 .withResultDetails(RESULT_DETAILS)
                 .build();
         AttemptAuditEventResult source = makeAttemptAuditEventResult(rsed);
+        underTest = new AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter(createMockUtilizer(ResultServiceEventData.class));
 
         AuditProto.AttemptAuditEventResult target = underTest.convert(source);
         assertGeneric(target);
-        assertThat(target.getEventTypeCase()).isEqualTo(AuditProto.AttemptAuditEventResult.EventTypeCase.RESULTSERVICEEVENTDATA);
-        assertThat(target.getResultServiceEventData().getResourceCrnList()).hasSameElementsAs(CRNS);
-        assertThat(target.getResultServiceEventData().getResultDetails()).isEqualTo(RESULT_DETAILS);
+        verify(mockAuditEventDataUtilizer, times(1)).modifyAuditEventBuilderWithEventData(any(), any());
     }
 
     @Test
     void convertWithResultApiRequestData() {
         ResultApiRequestData rsed = ResultApiRequestData.builder().withResponseParameters(RESPONSE_PARAMETERS).build();
         AttemptAuditEventResult source = makeAttemptAuditEventResult(rsed);
+        underTest = new AttemptAuditEventResultToGrpcAttemptAuditEventResultConverter(createMockUtilizer(ResultApiRequestData.class));
 
         AuditProto.AttemptAuditEventResult target = underTest.convert(source);
         assertGeneric(target);
-        assertThat(target.getEventTypeCase()).isEqualTo(AuditProto.AttemptAuditEventResult.EventTypeCase.RESULTAPIREQUESTDATA);
```

Comment:
I miss these tests in for the recently refactored converter classes

Reply:
because the mock does the actual data manipulation this test case line is not relevant anymore